proposals/reduce_spatial.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "reduce_spatial",
+    "summary": "Reduce horizontal spatial dimensions",
```

Comment:
I understand that "horizontal" means "x" and "y" in GIS-speak, but this might still be confusing to non-experts. Maybe just say "Reduce spatial dimensions 'x' and 'y'"

Reply:
Makes sense, changed.